### PR TITLE
Enhance show perecent

### PR DESCRIPTION
**Pull request by jperezsorrosal**
# Description of the changes
Improve the message of the analytics with a percentage

### Mentions

Person responsible for reviewing: @maleb0lge

### CODE REVIEW

File: TextAnalytics.ipynb [hunk 1]
```diff
@@ -1 +1,514 @@
...Qaw56EQvYIAiCIAiCIBgB4Q8cBMMnFrBBEARBEARBEATBVDCn6Q4EQRAEQRAEQRAEQQ6xgA2CIAiCIAiCIAimgljABkEQBMGIkHSUpPsk3S3pTknbSHqvpNWa7lsQBEEQTCNRAxsEQRAEI0DSS4DPAjuY2RJJ6wBPAm4EtjKzRxrtYBAEQRBMIbEDGwRBEASj4RnAI2a2BCAtWPcAngnMlzQfQNJrJN0k6XZJ50mam57/maSPp9/dJmmepMsl/VjSwek1z5B0XdrdvVfSy5r5qEEQBEEwHmIHNgiCIAhGQFqILgBWA64CzjGzayX9jLQDm3ZlvwXsbGa/lXQEsLKZHZted7yZnSLpc8BOwHbAKsB9ZvY0Se8HVjGz4yStAKxmZovH/mGDIAiCYEys2HQHgiAIgmA2YmaPSdoSeBmwI3COpCO7XrYtsBlwgyTwFOOb2n5/Ufp5DzA3LU4XS/qDpLWAW4F/kbQS8G0zu3N0nygIgiAImicWsEEQBEEwIszsT8A1wDWS7gEO6HqJgCvNbO8Z3mJJ+vnntv8Xj1c0s+skvRx4A3CmpE+Z2VeH9gGCIAiCYMKIGtggCIIgGAGSNpH0vLantgB+DiwGVk/PLQS2k/Tc1GY1Sc8vcYz1gYfM7MvAV4B5Q+l8EARBEEwosQMbBEEQBKNhLnBySvV9AvhP4G+BvYFLJT1oZjtKehvwdUkrp3YfBv4j8xg7AIdLehx4DNh/iP0PgiAIgokjRJyCIAiCIAiCIAiCqSBSiIMgCIIgCIIgCIKpIBawQRAEQRAEQRAEwVQQC9ggCIIgCIIgCIJgKogFbBAEQRAEQRAEQTAVxAI2CIIgCIIgCIIgmApiARsEQRAEQRAEQRBMBbGADYIgCIIgCIIgCKaCgTCm0AAAAAZJREFU/wNAv1CB5oDHLgAAAABJRU5ErkJggg==\n","text/plain":["<Figure size 1152x648 with 1 Axes>"]},"metadata":{"tags":[],"needs_background":"light"}}]},{"metadata":{"trusted":false,"id":"3Z0P4Lg6ih31","colab_type":"code","colab":{}},"cell_type":"code","source":[""],"execution_count":0,"outputs":[]}]}
```
Comment (maleb0lge) [inline TextAnalytics.ipynb:1]: We can't really compare this files, the original one is not formatted

File: TextAnalytics.ipynb [hunk 2]
```diff
@@ -1 +1,514 @@
...
+    "    jsonData = response.read().decode(\"UTF-8\")\n",
+    "    data = json.loads(jsonData)\n",
+    "    print('Sentiment goes from 0 to 1, where 0 is super negative and 1 is super positive.')\n",
+    "    print('------------------------------------------------------------------------------')\n",
+    "    print('')\n",
+    "    for document in data['documents']:\n",
+    "        #sentiment = \"positive\"\n",
+    "        #if document['score'] <= 0.5:\n",
+    "        #    sentiment = \"negative\"\n",
+    "        print('Document ' + document['id'] + ' has a ' + str(document['score']) + '/1.00 sentiment')\n",
```
Comment (jperezsorrosal) [inline TextAnalytics.ipynb]: I forgot to remove the original line of the message

File: TextAnalytics.ipynb (hunk 1 above)
Comment (maleb0lge) [reply TextAnalytics.ipynb:1]: Oh I see, I could see it and I saw your change but you forgot to remove the original print message.

**Review by maleb0lge — APPROVED** [review]
Now everything seems ok

**Pull request merged**